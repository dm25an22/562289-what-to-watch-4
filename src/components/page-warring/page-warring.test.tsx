import * as React from 'react';
import PageWarring from './page-warring';
import * as renderer from "react-test-renderer";

const MockChildren = () => {
  return (
    <div />
  );
};

it(`render PageWarring component`, () => {
  const tree = renderer.create(
      <PageWarring />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

it(`render PageWarring component with children`, () => {
  const tree = renderer.create(
      <PageWarring>
        <MockChildren />
      </PageWarring>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
