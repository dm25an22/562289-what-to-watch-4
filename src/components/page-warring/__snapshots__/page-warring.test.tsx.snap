// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PageWarring component 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "backgroundImage": "linear-gradient(-180deg,#180202 0%,#000 100%)",
      "color": "#dfcf77",
      "display": "flex",
      "flexDirection": "column",
      "fontFamily": "\\"Arial\\", \\"Helvetica\\", \\"sans-serif\\"",
      "height": "100vh",
      "justifyContent": "center",
      "width": "100vw",
    }
  }
/>
`;

exports[`render PageWarring component with children 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "backgroundImage": "linear-gradient(-180deg,#180202 0%,#000 100%)",
      "color": "#dfcf77",
      "display": "flex",
      "flexDirection": "column",
      "fontFamily": "\\"Arial\\", \\"Helvetica\\", \\"sans-serif\\"",
      "height": "100vh",
      "justifyContent": "center",
      "width": "100vw",
    }
  }
>
  <div />
</div>
`;
